### ISSUE TITLE
Sort modules from market search results by PG/CPU fitting cost.

### ISSUE THREAD

**Issue by vectorman1** [pyfa-org/Pyfa#758]
I feel like this would be a really neat feature to be able to sort from [here ](http://i.imgur.com/CeBTuda.png).

**Comment by blitzmann** [comment]
I agree, but IIRC sorting an wx.ListCtrl (the table of items there) isn't trivial. I will take another look at this as time allows. :)

**Comment by vectorman1** [comment]
If it's anything too tedious to do, maybe find a way to hack into it's native sorting? It surely does sort somehow, if its alphabetical you can put some bytes in the beginning to make them sort ?

**Comment by blitzmann** [comment]
IIRC, it sorts right now first by group, then by meta. That groups up like items together, and then sorts it by meta. This is the best way to sort the results by default. We do have the ability to do sorting when you click on the column, but I never got it to work correctly last time I tried.

**Comment by AurumGallente** [comment]
Icons are clickable but it doesn't sort items by values.
